src/main/resources/assets/wurst/lang/ja_jp.json

```diff
@@ -0,0 +1,139 @@
+    "description.wurst.hack.liquids": "ブロックを液体に設置できるようにします。",
+    "description.wurst.hack.lsd": "幻覚を引き起こします。",
+    "description.wurst.hack.masstpa": "すべてのプレーヤーにTPAリクエストを送信します。\n誰かが受け入れると停止します。",
+    "description.wurst.hack.mileycyrus": "twerkになります。",
+    "description.wurst.hack.mobesp": "近くのMobをハイライトします。",
+    "description.wurst.hack.mobspawnesp": "モブがスポーンできるエリアをハイライトします。\n§eyellow§r - モブは夜にスポーンできます\n§cred§r - モブはいつでもスポーンできます",
+    "description.wurst.hack.multiaura": "一度に複数のエンティティを攻撃するより高速なKillaura。",
+    "description.wurst.hack.nameprotect": "すべてのプレイヤー名を非表示にします。",
+    "description.wurst.hack.nametags": "名前タグのスケールを変更して、いつでも読むことができるようにします。 また、こっそりプレイヤーの名前タグを見ることができます。",
+    "description.wurst.hack.navigator": "時間の経過とともに好みを学習する検索可能なGUI。",
+    "description.wurst.hack.nobackground": "インベントリーの背後にある暗い背景を削除します。",
+    "description.wurst.hack.noclip": "ブロック内を自由に移動できます。\nブロック（砂など）をアクティブにするには、頭に落とす必要があります。\n\n§c§l警告:§r ブロックを移動中にダメージを受けます！",
+    "description.wurst.hack.nofall": "落下によるダメージからあなたを守ります。",
+    "description.wurst.hack.nofireoverlay": "火がついているときにオーバーレイを表示しません。\n\n§c§l警告:§r これにより、気付かずに燃え尽きる可能性があります。",
+    "description.wurst.hack.nohurtcam": "ダメージをくらったときの揺れ効果を無効にします。",
+    "description.wurst.hack.nooverlay": "水と溶岩のオーバーレイを表示しません。",
+    "description.wurst.hack.nopumpkin": "頭にカボチャをかぶったときにオーバーレイを表示しません",
+    "description.wurst.hack.noslowdown": "ハチミツ、ソウルサンド、アイテムの使用によるスロー効果を解除します。",
+    "description.wurst.hack.noweather": "クライアント側の天気、時間、月の満ち欠けを変更できます。",
+    "description.wurst.hack.noweb": "蜘蛛の巣によって遅くなるのを防ぎます。",
+    "description.wurst.hack.nuker": "あなたの周りのブロックを自動的に壊します。",
+    "description.wurst.hack.nukerlegit": "すべてのアンチチートプラグインをバイパスする遅いNuker。\n通常のNoCheat +サーバーでは必要ありません！",
+    "description.wurst.hack.openwateresp": "「オープンウォーター」で釣りをしているかどうかを示し、オープンウォーターの計算に使用される領域の周りにボックスを描画します。",
+    "description.wurst.hack.overlay": "ブロックをマイニングするたびにNukerアニメーションをレンダリングします。",
+    "description.wurst.hack.panic": "有効なすべてのハックを即座にオフにします。\nこれに注意してください！",
+    "description.wurst.hack.parkour": "ブロックの端に到達したときに自動的にジャンプします。\nパルクールやジャンプ'n'ランに便利です。",
+    "description.wurst.hack.playeresp": "近くのプレイヤーをハイライトします。\n友達のESPボックスは青色で表示されます。",
+    "description.wurst.hack.playerfinder": "雷雨の際に遠くのプレイヤーを見つけます。",
+    "description.wurst.hack.portalgui": "ポータルでGUIを開くことができます。",
+    "description.wurst.hack.potionsaver": "あなたが静止している間、すべてのポーション効果を凍結します。",
+    "description.wurst.hack.prophuntesp": "Prophuntで偽のブロックを表示できます。\nMineplexProphuntのために作られました。 他のサーバーでは動作しない可能性があります。",
+    "description.wurst.hack.protect": "最も近いエンティティを追跡し、他のエンティティから保護するボット。\nUse .protectは、最も近いエンティティではなく特定のエンティティを保護します。",
+    "description.wurst.hack.radar": "近くのエンティティの場所を表示します。\n§cred§r - プレイヤー\n§6orange§r - モンスター\n§agreen§r - 動物\n§7gray§r - その他",
+    "description.wurst.hack.rainbowui": "§cM§aa§9k§ce§as §9e§cv§ae§9r§cy§at§9h§ci§an§9g §cc§ao§9l§co§ar§9f§cu§al§9.",

```

Comment:
This description is still in English. Please translate it to Japanese.

(It says "Makes everything colorful." with each character encoded in a different color. The `§c`, `§a`, `§9` are color codes, the rest is text.)